extensions/librdkafka/PublishKafka.h

```diff
@@ -28,6 +28,7 @@
 #include "core/Property.h"
 #include "core/logging/LoggerConfiguration.h"
 #include "rdkafka.h"
+#include <regex>
```

Comment:
This does not work with all variants of GCC that we support. Therefore you will either need to switch to regex.h ( C variant ) or put a note in the ReadMe that not all versions of gcc ( I don't recall, but remember the issues with GetFile ) may support this processor. Will then require some checks in bootstrap to avoid users building this suite of processors for those versions. When I have some time and am back home I can attempt a build across the variants. I am leaning toward 4.8.4 having issues, so I will attempt that first.

Reply:
I believe GCC prior to 4.9 didn't support C++11 regex. I'm leaning towards not supporting older compilers for this processor but can look into using regex.h instead if desired.

Reply:
I'm fine with limiting it to 4.9 forward.

Reply:
@dtrodrigues Did you need help with limiting to 4.9 insofar that we can limit it in the cmake and documentation?